### ISSUE TITLE
Aimeos Plugin crashing WHOLE caching mechanism for all pages

### ISSUE THREAD

**Issue by kledo-34** [lochmueller/staticfilecache#236]
Hey there,

if this is not a bug but just me not being able to configure things correctly, sorry for that, but now the following:

I have a site with Staticfilecache generally working correctly, but if I open a site with either a form plugin (with honeypot enabled) or a page with an aimeos plugin, all other pages that are opened from that point onwards show the message that they are not cachable anymore, although they were cachable before and although typo3 still caches them correctly.

More detailed:
Expected behaviour: In a different browser someone opens a page in the frontend and staticfilecache caches the page if possible, if not it gives an error message why it's not cachable. (e.g. my search page with ke_search is not cachable which is expected behaviour). But if I once opened a page with Aimeos Shop plugins or the form plugin (with honeypot enabled) staticfilecache will not cache any further pages. All pages that are requested to be cached after that show the error that they are not cachable. Flushing caches, dumping autoload etc. doesn't bring back functionality, I have to return to my backup. If I disable caching in the page settings of those problem pages, this still happens. 
So this bug makes it impossible for me to use Aimeos shop and staticfilecache together. 
If I disable honeypot for the form extension, it solves the problem for that plugin, but not for aimeos, I just refered to it as both show the same behaviour.

Again I expected that staticfilecache doesn't cache the shop pages, that's logic, but it doesn't cache any other page after that, which is a huge problem to me.

If this was already discussed somewhere or you have a solution, I would be really happy.
I don't even have a workaround yet.

**Environment**
- TYPO3 version(s): 9.5.11
- staticfilecache version: 10.1.0
- Is your TYPO3 installation set up with Composer (Composer Mode): no
- OS: Windows 10, (Wamp Server, modules activated and working)

**Comment by lochmueller** [comment]
Do you know this:
https://forge.typo3.org/issues/83212
https://forge.typo3.org/projects/typo3cms-core/repository/revisions/d3cae300dd5e9842dfec89e75f64241e5873d1d4/diff/typo3/sysext/form/Documentation/FAQ/Index.rst
https://forge.typo3.org/projects/typo3cms-core/repository/revisions/d3cae300dd5e9842dfec89e75f64241e5873d1d4/diff/typo3/sysext/form/Documentation/Config/proto/formElements/formElementTypes/Form/renderingOptions/honeypot/enable.rst

Is this the problem?

**Comment by kledo-34** [comment]
Dang yes, it is. I checked a lot of things but not the cookies in my frontend browser session although this is quite obvious. When I clear the browser cache over there everything gets cached again. 
As I wrote before, sorry for reporting, it is not a bug, just me not being smart enough :/

Thank you so much for taking the time and answering that, it solves the "problem".
And thanks a lot for developing such a great performance extension :)
